csharp/src/Ice/OutgoingResponseFrame.cs

```diff
@@ -201,16 +239,44 @@ public OutgoingResponseFrame(IncomingRequestFrame request, RemoteException excep
             IsSealed = true;
         }
 
-        private OutgoingResponseFrame(IncomingRequestFrame request, List<ArraySegment<byte>>? data = null)
+        private static (OutgoingResponseFrame ResponseFrame, OutputStream Ostr) PrepareReturnValue(
+            Current current,
+            FormatType? format)
+        {
+            var response = new OutgoingResponseFrame(current.Protocol,
+                                                     current.Encoding,
+                                                     current.IncomingRequestFrame.CompressionStatus);
+
+            // Always OK or Success (i.e. 0) followed by encapsulation.
+            byte[] buffer = new byte[256];
+            Debug.Assert((byte)ReplyStatus.OK == 0 && (byte)ResultType.Success == 0);
```

Comment:
Why do we have this Assert?

Reply:
It's just to show that the `0` below means both ReplyStatus.OK and ResultType.Success.

Reply:
Fixed!